recipes/qt/6.x.x/conanfile.py

```diff
@@ -402,7 +404,9 @@ def source(self):
 
         tools.replace_in_file(os.path.join("qt6", "CMakeLists.txt"),
                         "enable_testing()",
-                        "include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)\nconan_basic_setup(KEEP_RPATHS)\n"
+                        "include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)\n"
+                               "list(REMOVE_ITEM CONAN_INCLUDE_DIRS \"${CONAN_INCLUDE_DIRS_STRAWBERRYPERL}\")\n" # strawberryperl exports includes for jpg, png and many other dependencies that might interfere with the build
```

Comment:
I'm not sure why strawberryperl exports this include directory which contains its bundled headers for jpeg, png, pcre, postgres etc, but the package is used in many other places so not super keen on changing that...

Reply:
Please try https://github.com/conan-io/conan-center-index/pull/9288

Reply:
That seems to work! The generated conanbuildinfo.cmake for the Qt build says `set(CONAN_INCLUDE_DIRS_STRAWBERRYPERL )` which should mean it is the same as what I did here.

Reply:
ok, don't hesitate to leave a positive review on the PR then :)